doc/managing-wallets.md

```diff
@@ -120,4 +120,29 @@ After that, `getwalletinfo` can be used to check if the wallet has been fully re
 $ bitcoin-cli -rpcwallet="restored-wallet" getwalletinfo
 ```
 
-The restored wallet can also be loaded in the GUI via `File` ->`Open wallet`.
\ No newline at end of file
+The restored wallet can also be loaded in the GUI via `File` ->`Open wallet`.
+
+## Migrating Legacy Wallets to Descriptor Wallets
```

Comment:
Thanks for adding documentation here.

nit: instead of duplicating the text in the release notes, maybe we could link to this document? Also, we could link to this document from places were we say BDB is legacy (such as in build-UNIX.md for instance)?

Reply:
Done